pkg/playground/speed.js

```diff
@@ -28,11 +28,11 @@ function update() {
         proc = cockpit.script("echo $PPID && cat /proc/$PPID/statm");
         proc.then(function(data) {
             const parts = data.split("\n");
-            pid.innerHTML = parts[0];
-            memory.innerHTML = parts[1];
+            pid.textContent = parts[0];
+            memory.textContent = parts[1];
             proc = null;
         }, function(ex) {
-            memory.innerHTML = String(ex);
+            memory.textContent = String(ex);
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19047-20230630-121653-41e6f122-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)